learn/api-docs/ballerina/kafka/clients/Consumer.html

```diff
@@ -754,7 +754,7 @@ <h2>Remote Methods</h2>
                                 <tr>
                                     <td class="module-title truncate functions" title="getSubscription"><a
                                             class="functions" href="#getSubscription">getSubscription</a></td>
-                                    <td class="module-desc"><p>Returns set of topics wich are currently subscribed by the consumer.</p>
+                                    <td class="module-desc"><p>Returns set of topics which are currently subscribed by the consumer.</p>
```

Comment:
Thanks for taking the time and interest to send in a fix for this! However, the API docs are generated from the doc comments in the source code itself. So the actual fix should be done in the following file: https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

Can you please send resend the fix to the `ballerina-1.0.x` branch in the https://github.com/ballerina-platform/ballerina-lang repo?